ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/activeMapping/ContinuousHikingStateMachine/ReadyToPlanState.java

```diff
@@ -263,4 +265,15 @@ public void addWayPointPoseToList(PoseListMessage poseListMessage)
       walkToGoalWayPointPoses.add(latestWayPoint);
       debugger.publishStartAndGoalForVisualization(continuousPlanner.getStartStancePose(), latestWayPoint);
    }
+
+   /**
+    * This allows the {@link ReadyToPlanState#walkToGoalWayPointPoses} to be cleared.
+    * This empties the list so the user can place a fresh goal that Continuous Hiking will use.
+    */
+   public void clearWayPointList(Empty emptyMessage)
```

Comment:
This subscribes to a message that can be sent to the user. The JavaDoc should explain what this does. The reason for adding this now is that if the user mis-clicks there was nothing to clear that goal and reset. This caused problems when trying to use Continuous Hiking from Python.